Public/js/profile-app.js

```diff
@@ -121,12 +123,83 @@ $(document).ready(function() {
       html +=   '<td>';
       html +=     day;
       html +=   '</td>';
-      html +=   '<td><button class="btn btn-success">Remove</button></td>';
+      html +=   '<td><button class="btn btn-success remove">Remove</button></td>';
       html += '</tr>'
       $('#table').append(html);
     }
   };
 
+  Reservation.prototype.runRemove = function(button) {
+    $.ajax({
+      context: this,
+      type: 'DELETE',
+      url: 'reservations',
+      data: {
+        reservation: {
+          machine: this.machine,
+          hour: this.hour,
+          day: this.day
+        }
+      },
+      dataType: 'json',
+      success: function(response) {
+        console.log(response);
+        if(response.ok) {
+          $(button).parent().parent().remove();
+        } else if(response.authenticated == false) {
```

Comment:
I previously did it that way, but it doesn't work very well because if response.authenticated == undefined I don't want to execute that if look.